src/routes/(auth)/login/+server.ts

```diff
@@ -1,11 +1,12 @@
-import { TokenCreateDocument } from '$lib/graphql/schema';
+import { TokenCreateDocument, type UserError } from '$lib/graphql/schema';
```

Comment:
```suggestion
import { TokenCreateDocument } from '$lib/graphql/schema';
```

Please introduce `type` import separately